superpeer/handler/TimedResponseHandler.py

```diff
@@ -16,19 +17,65 @@ def serve(self, sd: socket.socket) -> None:
 		try:
 			command = sd.recv(4).decode()
 		except OSError as e:
-			shell_colors.print_red(f'\nUnable to read the command from the socket: {e}\n')
+			shell.print_red(f'\nUnable to read the command from the socket: {e}\n')
 			sd.close()
 			return
 
 		if command == "AQUE":
-			pass
+			try:
+				response = sd.recv(300).decode()
+			except socket.error as e:
+				shell.print_red(f'Unable to read the {command} response from the socket: {e}\n')
+				sd.close()
+				return
+			sd.close()
+
```

Comment:
Io gio questo blocco di codice lo sposterei sopra, precisamente sotto alla ricezione della command. In questo modo lo si scrive solo una volta